shared/fixes/ansible/mount_option_krb_sec_remote_filesystems.yml

```diff
@@ -0,0 +1,20 @@
+# platform = multi_platform_rhel, multi_platform_fedora
```

Comment:
It would be great if this module could do the changes via Ansible module `mount`. This [template](https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/templates/template_ANSIBLE_mount_option) for mount options can serve as inspiration

Reply:
Is there a reason for this to apply only for rhel and fedora? The bash remediation applies to all platforms.